apps/console/src/features/organizations/pages/organization-edit.tsx

```diff
@@ -53,6 +58,24 @@ const OrganizationEditPage: FunctionComponent<OrganizationEditPagePropsInterface
     const [ isAuthorizedOrganization, setIsAuthorizedOrganization ] = useState(false);
     const [ filterQuery, setFilterQuery ] = useState<string>("");
 
+    const { switchOrganization, switchOrganizationInLegacyMode } = useOrganizationSwitch();
+    const { legacyAuthzRuntime }  = useAuthorization();
+    const { onSignIn } = useSignIn();
+
+    const isFirstLevelOrg: boolean = useSelector((state: AppState) => state?.organization?.isFirstLevelOrganization);
+    const tenantDomain: string = useSelector((state: AppState) => state?.auth?.tenantDomain);
```

Comment:
We recently added the `useGetCurrentOrganizationType` hook to get the first level org and super org checks. Shall we use that hook here as well.

Example usage:

```js
const { isFirstLevelOrganization, isSuperOrganization } = useGetCurrentOrganizationType();

// use the above functions in code like this.
if (isFirstLevelOrganization() && isSuperOrganization()) {}
```

Reply:
Resolved with 773dd41164aa3f63112cf9182d16c6d1e7908f37